specification/cognitiveservices/Language.Conversations/main.tsp

```diff
@@ -586,29 +586,28 @@ model AnalyzeConversationJobsInput {
   @doc("Display name for the analysis job.")
   displayName?: string;
 
-  @added(Versions.v2023_04_01)
-  @doc("Analysis Input")
   @key
-  @visibility("read", "create")
+  @visibility("read", "create", "query" )
+  @doc("Analysis Input")
   analysisInput: MultiLanguageConversationAnalysisInput;
 
-  @added(Versions.v2023_04_01)
   @doc("Set of tasks to execute on the input conversation.")
-  tasks: Array<AnalyzeConversationLROTask>;
+  tasks: Array<AnalyzeConversationLroTask>;
 }
 
 #suppress "@azure-tools/typespec-azure-core/casing-style" "The names of Model types must use PascalCase"
 @doc("Base class for a long-running conversation input task.")
 @discriminator("kind")
 @added(Versions.v2023_04_01)
-model AnalyzeConversationLROTask extends TaskIdentifier {
+model AnalyzeConversationLroTask extends TaskIdentifier {
```

Comment:
The uppercase "LRO" should be correct. Language generators should do the right thing for generated code. Were you not seeing that, though?

Reply:
it was throwing a PascalCase warning. Ill suppress the warning.